# Trivial patch to remove a stale reference to send_http_header that is no
# longer available or necessary
# Closes Debian #507821
--- a/docs/api/ModPerl/Registry.pod
+++ b/docs/api/ModPerl/Registry.pod
@@ -31,7 +31,6 @@ For example:
 
   my $r = Apache2::RequestUtil->request;
   $r->content_type("text/html");
-  $r->send_http_header;
   $r->print("mod_perl rules!");
 
 XXX: STOPPED here. Below is the old Apache::Registry document which I
